CONTRIBUTING

```diff
@@ -1,59 +1,3 @@
 # Contributing to Eclipse Java development tools
 
-Thanks for your interest in this project.
-
-## Project description
-
-The JDT project provides the tool plug-ins that implement a Java IDE supporting
-the development of any Java application, including Eclipse plug-ins. It adds a
-Java project nature and Java perspective to the Eclipse Workbench as well as a
-number of views, editors, wizards, builders, and code merging and refactoring
-tools. The JDT project allows Eclipse to be a development environment for
-itself.
-
-* https://projects.eclipse.org/projects/eclipse.jdt
-
-## Developer resources
-
-Information regarding source code management, builds, coding standards, and
-more.
-
-* https://projects.eclipse.org/projects/eclipse.jdt/developer
-
-The project maintains the following source code repositories
-
-* http://git.eclipse.org/c/jdt/eclipse.jdt.core.binaries.git
-* http://git.eclipse.org/c/jdt/eclipse.jdt.core.git
-* http://git.eclipse.org/c/jdt/eclipse.jdt.debug.git
-* http://git.eclipse.org/c/jdt/eclipse.jdt.git
-* http://git.eclipse.org/c/jdt/eclipse.jdt.ui.git
-
-This project uses Bugzilla to track ongoing development and issues.
-
-* Search for issues: https://eclipse.org/bugs/buglist.cgi?product=JDT
-* Create a new report: https://eclipse.org/bugs/enter_bug.cgi?product=JDT
-
-Be sure to search for existing bugs before you create another one. Remember that
-contributions are always welcome!
-
-## Eclipse Contributor Agreement
-
-Before your contribution can be accepted by the project team contributors must
-electronically sign the Eclipse Contributor Agreement (ECA).
-
-* http://www.eclipse.org/legal/ECA.php
-
-Commits that are provided by non-committers must have a Signed-off-by field in
-the footer indicating that the author is aware of the terms by which the
-contribution has been provided to the project. The non-committer must
-additionally have an Eclipse Foundation account and must have a signed Eclipse
-Contributor Agreement (ECA) on file.
-
-For more information, please see the Eclipse Committer Handbook:
-https://www.eclipse.org/projects/handbook/#resources-commit
-
-## Contact
-
-Contact the project developers via the project's "dev" list.
-
-* https://dev.eclipse.org/mailman/listinfo/jdt-dev
\ No newline at end of file
+See https://github.com/eclipse-jdt/.github/blob/main/CONTRIBUTING.md
```

Comment:
is that file needed at all? when its deleted it shout be automatically taken from .github

Reply:
Removal will be a good choice.

Reply:
> is that file needed at all? when its deleted it shout be automatically taken from .github

I cannot say if there are legal requirements or alike and I also don't know if it's inherited. I just know that many, if not all, repos in the platform, equinox, PDE organization have it. Maybe just to avoid dead links?

But I removed it now.

Reply:
It was a honest question. I know from https://docs.github.com/de/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors that https://github.com/eclipse-jdt/eclipse.jdt.ui/contribute does link it. But i do not know who would know how to go to that URL, i don't see a link to it anywhere and have never used it myself